src/components/AdminDashboard.vue

```diff
@@ -28,16 +46,129 @@
 </template>
 
 <script>
-
- import PendingListingModule from "./PendingListingModule.vue";
+ import PendingListingModule from "./PendingListingModule.vue"
  import ContactInboxModule from "./ContactInbox.vue"
-export default {
-  name: "AdminDashboard",
-   
-  components: {
+ import Vue3ChartJs from '@j-t-mcc/vue3-chartjs'
```

Comment:
Not really major, but if you choose to use `;` be consistent and use for all lines.  Just a styling suggestion no changes needed

Reply:
^^ Same for `'` and `"`

Reply:
I would agree with this, like Justin said just a minor change, I have my prettier config do this so I don't have to worry about it.

Reply:
Noted, will do from now on. Thank you.